Test for DDH5 loader node fails

The following test fails
https://github.com/data-plottr/plottr/blob/e47c9941bc178a1d5f0d2e810776cacfd9f14132/test/pytest/test_ddh5.py#L84
```
FAILED                                    [100%]
test/pytest/test_ddh5.py:83 (test_loader_node)
qtbot = <pytestqt.qtbot.QtBot object at 0x121e2a7d0>

    def test_loader_node(qtbot):
        dds.DDH5Loader.useUi = False
    
        x = np.arange(3)
        y = np.repeat(np.linspace(0, 1, 5).reshape(1, -1), 3, 0)
        z = np.arange(y.size).reshape(y.shape)
    
        data = dd.DataDict(
            x=dict(values=x, unit='A',
                   __info__='careful!',
                   __moreinfo__='more words in here'),
            y=dict(values=y, unit='B'),
            z=dict(values=z, axes=['x', 'y'], unit='C'),
            __desc__='some description',
        )
        assert data.validate()
        dds.datadict_to_hdf5(data, FN, append_mode=dds.AppendMode.new)
        assert _clean_from_file(dds.datadict_from_hdf5(FN)) == data
    
        fc = linearFlowchart(('loader', dds.DDH5Loader))
        node = fc.nodes()['loader']
    
        assert fc.outputValues()['dataOut'] is None
    
        node.filepath = FN
>       out = fc.outputValues()['dataOut'].copy()
E       AttributeError: 'NoneType' object has no attribute 'copy'

test_ddh5.py:109: AttributeError
```

--- a/plottr/data/datadict_storage.py
+++ b/plottr/data/datadict_storage.py
@@ -255,7 +255,7 @@ def datadict_from_hdf5(basepath: str, groupname: str = 'data',
 
         grp = f[groupname]
         keys = list(grp.keys())
-        lens = [len(grp[k].value) for k in keys]
+        lens = [len(grp[k][:]) for k in keys]
 
         if len(set(lens)) > 1:
             if not ignore_unequal_lengths:
@@ -285,7 +285,7 @@ def datadict_from_hdf5(basepath: str, groupname: str = 'data',
                     entry['unit'] = deh5ify(ds.attrs['unit'])
 
                 if not structure_only:
-                    entry['values'] = ds.value[startidx:stopidx]
+                    entry['values'] = ds[startidx:stopidx]
 
                 # and now the meta data
                 for attr in ds.attrs:

--- a/test/pytest/test_ddh5.py
+++ b/test/pytest/test_ddh5.py
@@ -6,7 +6,7 @@ from plottr.data import datadict as dd
 from plottr.data import datadict_storage as dds
 from plottr.node.tools import linearFlowchart
 
-FN = './test_ddh5_data.dd.h5'
+FN = './test_ddh5_data.ddh5'
 
 
 def _clean_from_file(datafromfile):

